modules/project_cleanup/variables.tf

```diff
@@ -25,6 +25,18 @@ variable "organization_id" {
   description = "The organization ID whose projects to clean up"
 }
 
+variable "clean_up_org_level_cai_feeds" {
+  type        = bool
+  description = "Clean up organization level CAI Feeds."
+  default     = false
+}
+
+variable "target_included_feeds" {
+  type        = list(string)
+  description = "List of organization feeds that should be deleted. Regex example: `.*/feeds/fd-cai-monitoring-.*` "
+  default     = ["fd-cai-monitoring-"]
```

Comment:
this is not a valid default value for this case.
The default value should be `[]`